frontend/src/component/changeRequest/ChangeRequestStatusBadge/ChangeRequestStatusBadge.tsx

```diff
@@ -66,10 +67,21 @@ export const ChangeRequestStatusBadge: VFC<IChangeRequestStatusBadgeProps> = ({
                 ) : (
                     <Info fontSize={'small'} />
                 );
+            const scheduledAt = new Date(
+                schedule?.scheduledAt || '',
+            ).toLocaleString();
+
+            const tooltipTitle =
+                schedule?.status === 'pending'
```

Comment:
why is schedule optional here?

Reply:
It's not really optional when the state is Scheduled. Just a result of the type - will improve it with a union